developer-guide.md

```diff
@@ -169,3 +179,34 @@ The [developer console](https://developers.chrome.com/origintrials/#/trials/acti
   - User Subset: A small percentage of Chrome users will always be excluded from the trial, even
       when a valid token is provided. The exclusion percentage varies for each trial, but is
       typically less than 5%.
+  ### 21. What does the _Valid Until_ date mean for my tokens?
```

Comment:
I suggest defining an explicit anchor, rather than relying on the auto-generated one.
```suggestion
  ### 21. What does the _Valid Until_ date mean for my tokens?  <a name="valid-until"/>
```